### ISSUE TITLE
Fails to clean non-empty directory

### ISSUE THREAD

**Issue by belicekm** [incinity/gulp-invipo-deploy#1]
The clean operation sometimes fails when there is non-empty directory (with at least one sub-directory) on the server.

There is usually one of the following error messages:
Error: Directory not empty.
Error: Directory not found.

```
events.js:141
[22:24:51]      throw er; // Unhandled 'error' event
[22:24:51]      ^
[22:24:51]
[22:24:51]Error: Directory not empty.
[22:24:51]    at makeError (C:\TeamCity\buildAgent\work\ed93d5dc9ad927a3\node_modules\gulp-invipo-deploy\node_modules\vinyl-ftp\node_modules\ftp\lib\connection.js:1067:13)
[22:24:51]    at Parser.<anonymous> (C:\TeamCity\buildAgent\work\ed93d5dc9ad927a3\node_modules\gulp-invipo-deploy\node_modules\vinyl-ftp\node_modules\ftp\lib\connection.js:113:25)
[22:24:51]    at emitTwo (events.js:87:13)
[22:24:51]    at Parser.emit (events.js:172:7)
[22:24:51]    at Parser._write (C:\TeamCity\buildAgent\work\ed93d5dc9ad927a3\node_modules\gulp-invipo-deploy\node_modules\vinyl-ftp\node_modules\ftp\lib\parser.js:59:10)
[22:24:51]    at doWrite (_stream_writable.js:300:12)
[22:24:51]    at writeOrBuffer (_stream_writable.js:286:5)
[22:24:51]    at Parser.Writable.write (_stream_writable.js:214:11)
[22:24:51]    at TLSSocket.ondata (C:\TeamCity\buildAgent\work\ed93d5dc9ad927a3\node_modules\gulp-invipo-deploy\node_modules\vinyl-ftp\node_modules\ftp\lib\connection.js:273:20)
[22:24:51]    at emitOne (events.js:77:13)
[22:24:51]Process exited with code 1
```

**Comment by TomNovosad** [comment]
quick fox in c17e4dc

**Comment by TomNovosad** [comment]
quick fix in c17e4dc
